attributes/default.rb

```diff
@@ -34,3 +34,25 @@
   :minute => '33',
   :hour => '3'
 }
+
+#The following attributes are provided as examples. In almost every
+#imaginable case you will want to replace some or all of these with
+#your own values.
+
+default[:chef_server_populator][:solo_org] = {
+  :org_name => 'inception_llc',
+  :full_name => 'Chef Inception Organization',
+  :validator_pub_key => 'validator_pub.pem'
+}
+
+default[:chef_server_populator][:solo_org_user] = {
+  :name => 'populator',
+  :first => 'Populator',
+  :last => 'User',
+  :email => 'pop@hw-ops.com',
```

Comment:
set `pop@hw-ops.com` to `pop@example.com` so if it ever actually does get used, it's using a reserved domain.